### PR TITLE
Geolocation lat long post

### PR DESCRIPTION
**Pull request by SimonTanner**
Successfully get's the current location of the user in the class object GetGeoLocation and then passes this to the parent BeeButton, which then passes this to the BeeApi. Branch had master merged into it and conflicts resolved prior to pull request.

### CODE REVIEW

**Review by FiddlersCode — CHANGES_REQUESTED** [review]
Can you remove the build files pls before we do any more reviewing?  :)

**Comment by SimonTanner** [comment]
Build files deleted as requested

File: BeeAppy/__tests__/GetGeoLocation.test.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+describe('GetGeoLocation', () => {
+  it('Gets a latitude and logitude coordinate', done => {
+    let getGeoLocation = new GetGeoLocation();
+    // expect.assertions(1);
+    return expect(getGeoLocation.returnLocation()).resolves.toContain("latitude");
+    done();
+
+  })
+})
+// it('Gets a latitude and logitude coordinate', done => {
```
Comment (FiddlersCode) [inline BeeAppy/__tests__/GetGeoLocation.test.js:13]: pls remove commented code :)

File: BeeAppy/src/screens/GeoMap.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+          <Text>Latitude: {this.state.latitude}</Text>
+          <Text>Longitude: {this.state.longitude}</Text>
+        {this.state.error ? <Text>Error: {this.state.error}</Text> : null}
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
```
Comment (FiddlersCode) [inline BeeAppy/src/screens/GeoMap.js]: refactor styles to stylesheet

File: BeeAppy/src/screens/Home.js [hunk 3]
```diff
@@ -12,6 +15,7 @@ export default class Home extends React.Component {
   };
   render() {
     const { navigate } = this.props.navigation;
```
Comment (FiddlersCode) [inline BeeAppy/src/screens/Home.js]: Can we remove rendering of current lat/long?  Don't think it's relevant to user :)

**Review by FiddlersCode — CHANGES_REQUESTED** [review]

File: BeeAppy/src/screens/Home.js (hunk 3 above)
Comment (SimonTanner) [reply BeeAppy/src/screens/Home.js]: Can we just merge first and change that after as it's useful for checking things actually work.

File: BeeAppy/__tests__/GetGeoLocation.test.js (hunk 1 above)
Comment (SimonTanner) [reply BeeAppy/__tests__/GetGeoLocation.test.js:13]: These are left in due to the test timing out so I wanted to come back to this and see if it's possible to make the test pass

File: BeeAppy/src/screens/GeoMap.js (hunk 2 above)
Comment (SimonTanner) [reply BeeAppy/src/screens/GeoMap.js]: GeoMap is a class inherited from the previous geolocation base branch that jess worked on and I thought it was best to have this for reference

File: BeeAppy/src/screens/Home.js (hunk 3 above)
Comment (FiddlersCode) [reply BeeAppy/src/screens/Home.js]: ok cool :)

**Review by FiddlersCode — APPROVED** [review]

**Pull request merged**